gradle/libs.versions.toml

```diff
@@ -137,6 +139,8 @@ test-core-ktx = { group = "androidx.test", name = "core-ktx", version.ref = "and
 testng = { group = "org.testng", name = "testng", version.ref = "testng" }
 androidx-ui-test-junit4-android = { group = "androidx.compose.ui", name = "ui-test-junit4-android", version.ref = "uiTestJunit4Android" }
 androidx-runtime-livedata = { group = "androidx.compose.runtime", name = "runtime-livedata", version.ref = "runtimeLivedata" }
+androidx-junit-ktx = { group = "androidx.test.ext", name = "junit-ktx", version.ref = "junitKtx" }
+ui-test-junit4-android = { group = "androidx.compose.ui", name = "ui-test-junit4-android", version.ref = "uiTestJunit4AndroidVersion" }
```

Comment:
Conflict with main branch: Unresolved merge between `androidx-junit-ktx`, `ui-test-junit4-android`, and `androidx-navigation-runtime-ktx` as well as calendar-related dependencies. Please ensure to merge both testing and navigation dependencies. Don't forget to keep the calendar dependencies (`ical4j`, `compose`) intact (as the are needed for the calendar feature).